### PR TITLE
Fixes #18515: Parent breaks build

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/18515

### CODE REVIEW

File: webapp/sources/api-doc/paths/nodes/status.yml [hunk 1]
```diff
@@ -1,18 +1,11 @@
 # SPDX-License-Identifier: CC-BY-SA-2.0
 # SPDX-FileCopyrightText: 2013-2020 Normation SAS
 get:
-  summary: Get status (pending, accepted, unknown) of a list of nodes
-  description: 
+  summary: Get nodes acceptation status
+  description: Get acceptation status (pending, accepted, deleted, unknown) of a list of nodes
   operationId: getNodesStatus
   parameters:
-    name: "ids"
```
Comment (amousset) [inline webapp/sources/api-doc/paths/nodes/status.yml:8]: The problem was that parameters expects an array. I finally split it to allow easier reuse (and maybe better standardization of parameters).

File: webapp/sources/api-doc/components/parameters/node-ids.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: CC-BY-SA-2.0
+# SPDX-FileCopyrightText: 2013-2020 Normation SAS
+name: "nodeIds"
```
Comment (fanf) [inline webapp/sources/api-doc/components/parameters/node-ids.yml:3]: it expects `ids`, not `nodeIds`

Comment (fanf) [reply webapp/sources/api-doc/components/parameters/node-ids.yml:3]: should it be nodeIds ?

**Comment by amousset** [comment]
PR updated with a new commit

**Review by fanf — APPROVED** [review]

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**